### ISSUE TITLE
Enter should open/close select when it is focused

### ISSUE THREAD

**Issue by Mikilll94** [JedWatson/react-select#5154]
## Are you reporting a bug or runtime error?

https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html

According to a11y guidelines when the select control is focused it should be possible to open/close it by with Enter/Return key. Currently the Spacebar opens/closes select.

You can check this by yourself in this Code Sanbox example.

https://codesandbox.io/s/react-select-v5-sandbox-y5jtm
